worker/caasunitprovisioner/application_worker.go

```diff
@@ -98,8 +98,8 @@ func (aw *applicationWorker) loop() error {
 		brokerUnitsWatcher watcher.NotifyWatcher
 		brokerUnitsChannel watcher.NotifyChannel
 
-		appOperatorWatcher  watcher.NotifyWatcher
-		appOpertatorChannel watcher.NotifyChannel
+		appOperatorWatcher watcher.NotifyWatcher
+		appOperatorChannel watcher.NotifyChannel
```

Comment:
:+1:

Reply:
Opertator must be a kind of root vegetable. :potato: